### PR TITLE
make --config and --config-url work

### PR DESCRIPTION
**Pull request by drewfish**

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: bin/dep-audit [hunk 1]
```diff
@@ -22,7 +27,11 @@ var argv = require('yargs')
 .argv
 
 if (argv['config-url']) {
-  request(argv['config-url'], function (err, res, body) {
+  request.get({
+    method: 'GET',
```
Comment (sylvio) [inline bin/dep-audit]: by using request.get method is set to 'GET'. Even if it was something else it would be reset to 'GET'.

Comment (drewfish) [inline bin/dep-audit]: Thanks, I'll fix that (once I get the builds to pass).

**Comment by evantorrie** [comment]
LGTM 👍

**Pull request merged**